acrarium/src/main/kotlin/com/faendir/acra/service/MailService.kt

```diff
@@ -68,6 +69,9 @@ class MailService(
     @Value("\${server.context-path}")
     private val baseUrl: String? = null
 
+    @Value("\${spring.mail.senderAddress}")
```

Comment:
Please shorten to `sender` from `senderAdress`. I think that's clear enough and avoid casing issues.